ext/standard/basic_functions.stub.php

```diff
@@ -925,13 +939,17 @@ function stat(string $filename): array|false {}
 
 function lstat(string $filename): array|false {}
 
+/** @param string|int $user */
 function chown(string $filename, $user): bool {}
 
+/** @param string|int $group */
 function chgrp(string $filename, $group): bool {}
 
 #if HAVE_LCHOWN
+/** @param string|int $user */
 function lchown(string $filename, $user): bool {}
```

Comment:
This and `lchgrp ` return `false` on Windows. Could I yet add the `string|int` type declaration after fixing ZPP?

Reply:
I think the code is confusing here. This particular function just isn't defined on windows due to `#if HAVE_LCHOWN`.

Reply:
Oh, I missed the `#if HAVE_LCHOWN` around the stub. It explains everything then :) And then I'll add the proper type declaration and I'll get rid of the `return false`.

Reply:
Actually, I've just realized that the main problem was that I wasn't sure if `HAVE_LCHOWN` is false when `defined(WINDOWS)` is true.